app/dao/notifications_dao.py

```diff
@@ -282,6 +282,11 @@ def get_notifications_for_job(service_id, job_id, filter_dict=None, page=1, page
     return query.order_by(asc(Notification.job_row_number)).paginate(page=page, per_page=page_size)
 
 
+@statsd(namespace="dao")
+def get_notification_count_for_job(service_id, job_id):
+    return Notification.query.filter_by(service_id=service_id, job_id=job_id).count()
```

Comment:
I checked as I was curious and this can actually use the two [separate indexes](https://use-the-index-luke.com/sql/explain-plan/postgresql/operations) on service_id and job_id:

```
Aggregate  (cost=43376.36..43376.37 rows=1 width=8)
  ->  Bitmap Heap Scan on notifications n  (cost=36194.63..43319.72 rows=22656 width=0)
        Recheck Cond: (job_id = '551beb20-3561-4051-b548-a7dab97f1b63'::uuid)
        Filter: (service_id = '1c31507f-a7ec-4102-9cbe-c648388c1e8a'::uuid)
        ->  Bitmap Index Scan on ix_notifications_job_id  (cost=0.00..36188.97 rows=49673 width=0)
              Index Cond: (job_id = '551beb20-3561-4051-b548-a7dab97f1b63'::uuid)
```

But most likely won't use these, as the jobs are 50K rows and too small, whereas an seq scan [will be less costly](https://www.gojek.io/blog/the-case-s-of-postgres-not-using-index).